### PR TITLE
Travis CI

### LINKED ISSUE #8: add Travis CI

### PR DESCRIPTION
**Pull request by semelianova**
close #8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,85 @@
-image: Rhoc
+language: go
+
+go:
+- 1.13
+
+before_install:
+  - if [ "$TRAVIS_OS_NAME" = "windows" ]; then choco install make; fi
+
+install:
```
Comment (vnlitvinov) [inline .travis.yml]: what's the difference between `install` and `before_install` sections?

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,85 @@
...
+  install: skip
+  if: tag IS present
+  script: make GOOS=$(go env GOOS)
+  deploy:
+    - provider: releases
+      token:
+        secure: $GITHUB_API_KEY
+      file_glob: true
+      file: package-*.tar.gz
+      skip_cleanup: true
```
Comment (vnlitvinov) [inline .travis.yml]: what does this do?

File: .travis.yml (hunk 1 above)
Comment (semelianova) [reply .travis.yml]: As far as I understand `before_install` is usually used to install common packages that aren't included in the standard distributions or to reinstall specific versions, `install` is used to install third-party dependencies which are used for building or testing.

File: .travis.yml (hunk 2 above)
Comment (semelianova) [reply .travis.yml]: If `skip_cleanup` is false Travis deletes all artifacts which are created during the build.

File: .travis.yml (hunk 1 above)
Comment (vnlitvinov) [reply .travis.yml]: Still unclear to me, but it doesn't matter for now :)

**Pull request merged**